test/test_average_unpooling_layer.h

```diff
@@ -13,33 +13,35 @@
     names of its contributors may be used to endorse or promote products
     derived from this software without specific prior written permission.
 
-    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+   AND ANY
     EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
     WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-    DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY
+    DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+   FOR ANY
     DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
     (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
     LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
     ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-    (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+    (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+   THIS
     SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 #pragma once
- #include "gtest/gtest.h"
+#include "gtest/gtest.h"
 #include "testhelper.h"
 #include "tiny_dnn/tiny_dnn.h"
 
 namespace tiny_dnn {
 
-TEST(ave_unpool, gradient_check) { // sigmoid - cross-entropy
-    typedef cross_entropy loss_func;
+TEST(ave_unpool, gradient_check) {  // sigmoid - cross-entropy
+    typedef cross_entropy       loss_func;
     typedef activation::sigmoid activation;
     typedef network<sequential> network;
 
     network nn;
-    nn << fully_connected_layer<activation>(3, 4)
-        << average_unpooling_layer<activation>(2, 2, 1, 2) // 2x2 => 4x4
-        << average_pooling_layer<activation>(4, 4, 1, 2);
+    nn << fully_connected_layer<activation>(3, 4) << average_unpooling_layer<activation>(2, 2, 1, 2)  // 2x2 => 4x4
+       << average_pooling_layer<activation>(4, 4, 1, 2);
```

Comment:
Please add a line break between `fully_connected_layer<activation>(3, 4)` and `average_unpooling_layer<activation>(2, 2, 1, 2)`.